packages/wonder-stuff-core/src/kind-error.js

```diff
@@ -111,7 +111,7 @@ export class KindError extends Error {
             stripStackFrames,
             minimumFrameCount,
             compositeStack,
-        }: Options = {},
+        }: Options = Object.freeze({}),
```

Comment:
This was a new Flow error that popped up.  Not sure why it wasn't caught before.